tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -33,7 +33,7 @@ Target GenerateVariables
             void GenerateConditionVariables()
             {
                 GenerateConditionVariableBasedOnGitChange("isTracerChanged", new[] { "tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation", "tracer/src/Datadog.Tracer.Native" }, new string[] {  });
-                GenerateConditionVariableBasedOnGitChange("isDebuggerChanged", new[] { "tracer/src/Datadog.Trace/Debugger/Instrumentation", "tracer/src/Datadog.Tracer.Native" }, new string[] { });
+                GenerateConditionVariableBasedOnGitChange("isDebuggerChanged", new[] { "tracer/src/Datadog.Trace/Debugger/Instrumentation", "tracer/src/Datadog.Tracer.Native", "tracer/test/Datadog.Trace.Debugger.IntegrationTests" }, new string[] { });
```

Comment:
Should we look for changes in the other debugger test projects too?

Reply:
Do you mean `Samples.Probes`?
Probably we should.
@OmerRaviv ?

Reply:
Yeah great catch! We should.